multitenancy/tenant-manager-api/src/test/java/io/apicurio/multitenant/TenantManagerClientAuthTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.multitenant;
+
+import io.apicurio.multitenant.client.Auth;
+import io.apicurio.multitenant.client.TenantManagerClient;
+import io.apicurio.multitenant.client.TenantManagerClientImpl;
+import io.apicurio.registry.utils.tests.ApicurioTestTags;
+import io.apicurio.registry.utils.tests.AuthTestProfileWithoutRoles;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.keycloak.authorization.client.util.HttpResponseException;
+
+import javax.enterprise.inject.Typed;
+
+
+@QuarkusTest
+@TestProfile(AuthTestProfileWithoutRoles.class)
+@Tag(ApicurioTestTags.DOCKER)
```

Comment:
if you are adding this tag you need to configure `maven-surefire-plugin` in this projects pom.xml file the same way it's configured in `app` module, basically add this:
```
<build>
  <plugins>
    <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-surefire-plugin</artifactId>
                <configuration>
                    <groups>${groups}</groups>
                    <systemProperties>
                        <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
                    </systemProperties>
                </configuration>
            </plugin>
  </plugins>
</build>
```
```
<profiles>
        <profile>
            <id>no-docker</id>
            <properties>
                <groups>!docker</groups>
            </properties>
        </profile>
</profiles>
```